grammar.js

```diff
@@ -1296,15 +1246,13 @@ module.exports = grammar({
 
     where: $ => seq(
       $.keyword_where,
-      $.where_expression,
+      $._expression,
```

Comment:
Rather than have custom nodes for these, what do you think of using a named node, e.g. "expression" or "predicate"? Same for the former `join_expression', & 'order_expression', etc.

```suggestion
      field("expression", $._expression),
```

```suggestion
      field("predicate", $._expression),
```

Reply:
I like it for join and where, definitely; order is kind of the odd one out because it's a list (potentially _containing_ binary expressions) while the others are binary expression trees linked by conjunctions and disjunctions. This one went places, though, so I'll raise that separately.